rewrite-java-11/src/main/java/org/openrewrite/java/Java11ParserVisitor.java

```diff
@@ -1953,6 +1958,20 @@ private String skip(@Nullable String token) {
         return token;
     }
 
+    /**
+     * Advances the cursor if the current cursor position starts with the given token.
+     *
+     * @param token Token to skip
+     * @return true if the token is found, otherwise false.
+     */
+    private boolean skipIfPresent(String token) {
```

Comment:
`skip` already has this behavior i believe?
![image](https://user-images.githubusercontent.com/1697736/122259475-14373000-ce87-11eb-8dc4-209aba0f6d33.png)

Reply:
there is no way of knowing if the token was found/not found and I didn't want to change the existing contract.